### DISCUSSION TITLE
WebP Zero :Day

### DISCUSSION THREAD

**Discussion by 6Hat** [SixLabors/ImageSharp#2540] [Q&A] [answered]
In light of the recently revealed WebP Zero Day Vulnerability... how can we update the libwebp.dll files to the latest version of 1.3.2+?

**Comment by JimBobSquarePants** [comment, marked as answer]
We don’t use libwebp. See #2538

**Reply by 6Hat** [reply]
Aren't you heading Image Processor as well? Are they related libraries?

**Reply by JimBobSquarePants** [reply]
No they’re not. ImageProcessor is an old, obsolete, long retired, wrapper around System.Drawing with a few extras.

**Comment by 6Hat** [comment]
Ok... just now saw the link... thank you.
